lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -1313,6 +1314,10 @@ class DomCSSStyleSheet extends DomStyleSheet {}
 
 extension DomCSSStyleSheetExtension on DomCSSStyleSheet {
   external DomCSSRuleList get cssRules;
+  Iterable<DomCSSRule> get rules =>
+      createDomListWrapper<DomCSSRule>(js_util
+          .getProperty<_DomList>(this, 'cssRules'));
```

Comment:
I went back and forth on this one, and I don't know why I didn't do it (I guess there's a method that calls .length on this repeatedly). I don't think this will negatively impact performance though, so done.